Cargo.toml

```diff
@@ -64,3 +68,8 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [profile.dev]
 opt-level = 2
+
+# TODO: Point to a version tag once necessary features have stabilized
```

Comment:
If we're leaving this TODO in place, probably makes sense to open a linear ticket to help track it.